### PR TITLE
Cloud/Classic separate project initializer scripts + Dispatcher/Docker-Compose setups

### PR DESCRIPTION
**Pull request by krystian-panek-wttech**

### CODE REVIEW

File: project/cloud/aem/script/deploy.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+step "deploy AEM application"
+aem package deploy --file "all/target/*.all-*.zip"
+clc
+
+step "deploy AEM dispatcher"
+sh aem/home/opt/sdk/dispatcher/bin/validate.sh dispatcher/src > aem/home/var/log/dispatcher-validate.log
+clc
+docker tag "$(docker load --input "aem/home/opt/sdk/dispatcher/lib/dispatcher-publish-${ARCH}.tar.gz" | awk -v 'FS= ' '{print $3}')" "adobe/aem-ethos/dispatcher-publish:latest"
+clc
+docker compose up -d
```
Comment (krystian-panek-wttech) [inline project/cloud/aem/script/deploy.sh:19]: @maciej-majchrzak-wttech you probably like it ;) 

no Gradle wrappers around

File: project/classic/aem/script/undeploy.sh [hunk 2]
```diff
@@ -0,0 +1,3 @@
+step "undeploy AEM dispatcher"
```
Comment (maciej-majchrzak-wttech) [inline project/classic/aem/script/undeploy.sh:1]: This name is kinda funny and misleading too. Do we even need it? I'd move it to "down.sh" and get rid of undeploy

Comment (krystian-panek-wttech) [reply project/classic/aem/script/undeploy.sh:1]: I wanted to have things paired conceptually..

down is a more general hook, but yes, I have same doubt here; I will rethink it :)

Comment (krystian-panek-wttech) [reply project/classic/aem/script/undeploy.sh:1]: down was designed to control only aem; undeploy probably for the rest things; maybe naming could be polished somehow but I have no idea for now

**Pull request merged**